ntbs-service/deployments/live.yml

```diff
@@ -55,8 +58,30 @@ spec:
               value: ""
             - name: AppConfig__AuditingEnabled
               value: "false"
```

Comment:
Not part of these changes, but do we want this to be set to true by default?

Reply:
before launch - definitely, but whilst i'm using `live` for migration trials it's a bit easier with this off :)